pages/bookmarks.jsx

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
```

Comment:
does that not mean we can just re-use `<ProjectLoader>` but with an optional attribute `sortOnLoad=....` so that we don't need this separate component, but we can simply make ProjectLoader either force-sort if there is a `this.props.sortOnLoad`, or do what it normally does if there isn't?